rudder-web/src/main/scala/com/normation/rudder/web/rest/RestAPI.scala

```diff
@@ -119,4 +121,19 @@ case class APIDispatcher (
     }
 
 
+    serve {
+      case Get("api" :: "info" :: Nil,req) =>
+        implicit val action = "listChangeRequests"
+        implicit val prettify = restExtractor.extractPrettify(req.params)
+
+        val availableVersions = apisByVersion.keySet.map(_.value).toList.sorted
+        val max = availableVersions.max
+        import net.liftweb.json.JsonDSL._
+        val versions: JValue =
```

Comment:
I think we need to have versions { latest = ... , available } else it will be hard to extend the info in the future. 
Perhaps it's apiVersions ? version is shorter, thought